quic/README.md

```diff
@@ -0,0 +1,42 @@
+# QUIC in libp2p
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+|-----------------|---------------|--------|-----------------|
+| 1A              | Working Draft | Active | r1, 2022-12-30  |
```

Comment:
```suggestion
| 3A              | Recommendation | Active | r1, 2022-12-30  |
```

Given that this is widely deployed, `Recommendation` seems the most appropriate to me.